doc/platforms/x86_64_linux/x86_64_linux.md

```diff
@@ -73,14 +73,14 @@ If it succeeds, you can see the container runs as follows:
 **********************************
 5d2efd81057fe56236602acfece0e8f11d447b54627f4f3669b18c85a95b8687
 CONTAINER ID        IMAGE                      COMMAND             CREATED                  STATUS                  PORTS               NAMES
-5d2efd81057f        edge-orchestration:coconut  "sh run.sh"         Less than a second ago   Up Less than a second                       edge-orchestration
+5d2efd81057f        edge-orchestration:coconut "sh run.sh"         Less than a second ago   Up Less than a second                       edge-orchestration
 ```
 
 and the built image as follows:
 ```shell
 $ docker images
 REPOSITORY                 TAG                 IMAGE ID            CREATED             SIZE
-edge-orchestration         coconut              502e3c07b01f        3 seconds ago       132MB
+edge-orchestration         coconut             502e3c07b01f        3 seconds ago       132MB
```

Comment:
Please update the image size from 132MB to 185MB.

Reply:
Applied your comment! Thank you for the kind review! ^^